File: mytestvueapp.client/src/entities/Point.ts

export default class Point {
  //required
  id: number;
  latitude: number;
  longitude: number;
  artspaceId: number;
  title: string;
  isPublic: boolean;

  constructor() {
    this.id = 0;
    this.latitude = 38.897634;
    this.longitude = -77.036570;
    this.title = "";
    this.artspaceId = 0;
    this.isPublic = false;
  }
}

File: MyTestVueApp.Server/Entities/Point.cs
using MyTestVueApp.Server.Models;
using System.Reflection.Metadata.Ecma335;

namespace MyTestVueApp.Server.Entities
{
    public class Point
    {
        //Required
        public int Id { get; set; }
        public int ArtspaceId { get; set; }
        public string Title { get; set; }
        public decimal Latitude { get; set; }
        public decimal Longitude { get; set; }
    }
}
